### PR TITLE
Fixes #23891 - share all templates to all taxonomies

### PR DESCRIPTION
**Pull request by ares**
1.18 blocker, core seeds all templates to all taxonomies too

### CODE REVIEW

File: db/seeds.d/70-job_templates.rb [hunk 1]
```diff
@@ -1,8 +1,12 @@
+organizations = Organization.unscoped.all
```
[the remark this answers is not in the document]
Comment (ares) [reply db/seeds.d/70-job_templates.rb]: wait, what? no

**Comment by ares** [comment]
test failures related, I'll need to check

**Comment by ares** [comment]
failures don't seem to be related now, getting many `Sequel::DatabaseError: PG::DuplicateTable: ERROR:  relation "dynflow_execution_plans" already exists`

**Comment by iNecas** [comment]
@adamruzicka would you mind looking at the test failures errors? Seems related to https://github.com/theforeman/foreman/commit/e5b4634fa2372e28d2792418305f4205477071de

**Comment by adamruzicka** [comment]
[test]

**Comment by adamruzicka** [comment]
Tests running [here](http://ci.theforeman.org/job/foreman_remote_execution-pull-request/31/)

**Comment by iNecas** [comment]
[test foreman_remote_execution]

**Comment by adamruzicka** [comment]
The tests are finally green, merging

**Comment by adamruzicka** [comment]
Thanks @ares !

**Pull request merged**